src/HttpClient/GuzzleClient.php

```diff
@@ -88,11 +87,15 @@ public function sendRequest($method, string $url, array $defaultHeaders, $params
         $opts['headers'] = $defaultHeaders;
         $opts['params'] = $params;
 
-        [$rbody, $rcode, $rheader] = $this->_executeRequest($opts, $url);
```

Comment:
Seems like I have some unmerged changes carried over from version 2.4.1.

Reply:
Yup yup. Can you update master on your fork and merge master into your feature branch?

Reply:
Done, seems good to me :)